### PR TITLE
audit: placement enforcement tweaks

### PR DESCRIPTION
**Pull request by DomT4**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you successfully run `brew tests` with your changes locally?

-----

More style harmonisation.

CC @ilovezfs as you asked for the `include` enforcement.

### CODE REVIEW

File: Library/Homebrew/cmd/audit.rb [hunk 1]
```diff
@@ -203,6 +203,7 @@ def audit_file
     return unless @strict
 
     component_list = [
+      [/^  include Language::/,            "include method"],
```
Comment (UniqMartin) [inline Library/Homebrew/cmd/audit.rb]: I'm afraid `include method` is going to be confusing, as `foo method` tends to refer to `def foo` (see below). How do you feel about referring to it as simply `include` or maybe `include directive`?

**Comment by UniqMartin** [comment]
:+1: aside from some terminology nitpicking.

File: Library/Homebrew/cmd/audit.rb (hunk 1 above)
Comment (DomT4) [inline Library/Homebrew/cmd/audit.rb]: `include directive` sounds 👍 to me. Will change.

**Comment by ilovezfs** [comment]
Shouldn't this also force the blank line following?

**Comment by UniqMartin** [comment]
> ```
> audit: enforce include method placement
> audit: tweak include wording
> ```

Sorry if I'm being obnoxious here, but you are going to squash these two commits to a single commit “audit: enforce include directive placement” before pulling, aren't you?

> Shouldn't this also force the blank line following?

I'm pretty sure we don't have any infrastructure in place to enforce inclusion/omission of blank lines, but I'm sure @DomT4 is happy to make that part of this PR. :trollface:

**Comment by DomT4** [comment]
> Sorry if I'm being obnoxious here, but you are going to squash these two commits to a single commit “audit: enforce include directive placement” before pulling, aren't you?

Thought I already did, to be honest. Apparently not.

> I'm pretty sure we don't have any infrastructure in place to enforce inclusion/omission of blank lines

I can think of no obviously sane way to communicate placement of empty lines to people. We'd be using it already if I could think of a clever way to both implement & communicate it 😅.

**Comment by UniqMartin** [comment]
> I can think of no obviously sane way to communicate placement of empty lines to people.

How about “please add an empty line between include directive (line 2) and desc (line 3)” and “please remove the empty line(s) between desc (line 3) and homepage (line 5)”? Seems entirely doable to me on the communication level. As for the implementation side …

**Comment by ilovezfs** [comment]
I think some apparatus of blank-line virtue would be nice to have but it needn't block this PR despite how much I know @DomT4 is champing at the bit to code it.

**Pull request closed without merging**